src/routes/guides/fetching-data.mdx

```diff
@@ -109,7 +109,7 @@ function App() {
 
         <Switch>
           <Match when={user.error}>
-            <span>Error: {user.error()}</span>
+            <span>Error: {user.error.message}</span>
```

Comment:
otherwise breaks rendering with `Uncaught (in promise) TypeError: user.error is not a function`